Core/include/Acts/Propagator/MultiEigenStepperLoop.ipp

```diff
@@ -35,16 +35,14 @@ auto MultiEigenStepperLoop<E, R, A>::boundState(
       }
     }
 
-    if (failedBoundTransforms > 0) {
-      ACTS_ERROR("Multi component bound state: "
-                 << failedBoundTransforms << " of " << numberComponents(state)
-                 << " transforms failed");
-    }
-
     if (states.size() == 0) {
       return MultiStepperError::AllComponentsConversionToBoundFailed;
     }
 
+    if (failedBoundTransforms > 0) {
+      return MultiStepperError::SomeComponentsConversionToBoundFailed;
```

Comment:
So this doesn't log ERROR anymore, but newly returns an error, right?
Should this be an additional INFO or lower log message still to give more info if needed for debugging?

Reply:
So the problem is that there is no access to a logger anymore (since I removed the internal logger of the `MultiEigenStepper` ... I'm not sure if it would be better to change the general stepper interface to pass a `LoggerWrapper` instance to this function @paulgessinger ?

Reply:
I see. The main point of `LoggerWrapper` was that it can be easily defaulted, which then doesn't log anything. In any case, if we change this, it should be a separate PR I would say.